### PR TITLE
Migrated Scribe to a CARE App + Context

### LINKED ISSUE #8134: Make Scribe Aware of already present form data
Currently scribe is not aware of already existing form data, which might conflict with the newly generated data

### LINKED ISSUE #8227: Scribe is malfunctioning in iphone
Follow-up : https://github.com/coronasafe/care_fe/issues/8202 https://github.com/coronasafe/care_fe/issues/8203

**Describe the bug**

When the upload is completed for the scribe using iPhone, the whisper is failing by showing a error in log " AI form fill processing failed: Error code: 424 - {'error': {'message': "Invalid file format. Supported formats: ['flac', 'm4a', 'mp3', 'mp4', 'mpeg', 'mpga', 'oga', 'ogg', 'wav', 'webm']", 'type': 'invalid_request_error', 'param': None, 'code': None}} "

**To Reproduce**

1. Go to patient log update page 
2. Upload a audio for auto-filling using iphone and See error

**Expected behavior**

ensure the whisper is working properly and the expected audio is being received post the audio transcription 

**Screenshots**

### LINKED ISSUE #8285: Enhancement to the CARE Scribe
Follow-up : https://github.com/coronasafe/care_fe/pull/8183#issuecomment-2274437517

**Is your feature request related to a problem? Please describe.**

- [ ] he diagnosis is not being auto-filled based on the diagnosis code
Note: Auto-fill only works if we say the name of the diagnosis cholera, not the code 1A00
- [ ] We cannot discontinue a medicine using the scribe because the scribe is unaware of the currently filled data.
- [ ] The errors are not normal user-friendly messages for example whisper failed, error creating scribe instance, something went wrong

**Describe the solution you'd like**

- [ ] diagnosis autofill should be work based on code
- [ ] scribe should be aware of currently filled data to interact with existing filled data
- [ ] make the error message normal user friendly [ @shivankacker  can you push a fix  for this  alone First, before working on the other enhancements ]

### LINKED ISSUE #8580: Add Scribe support to newly added community log update fields
**Is your feature request related to a problem? Please describe.**

Currently, the newly added fields under the routine section of the community log update fields are not working in scribe auto-filling

**Describe the solution you'd like**

Add scribe support to the routine section of the community log update

**Screenshot**

image

### LINKED ISSUE #8821: Convert Scribe to Care App

### LINKED ISSUE #8978: Replace unnecessary range auto complete form fields in Daily Rounds with integer fields
Image

### PR DESCRIPTION
**Pull request by shivankacker**
## Proposed Changes

- Fixes #8134 
- Fixes #8285 
- Fixes #8978 
- Fixes #8821 
- Fixes #8227 
- Fixes #8580

Moves Scribe to https://github.com/ohcnetwork/care_scribe_fe

https://github.com/user-attachments/assets/8bbfe6d7-c8be-4a14-bdb5-30ccad3dc4e7

Requires care_scribe backend plugin pointing to https://github.com/ohcnetwork/care_scribe/pull/12

> Make sure to add the `SCRIBE_ENABLED` flag through Django Admin to your user

Requires the care_scribe_fe frontend appp. Add that by updating your env to include
`
REACT_ENABLED_APPS="ohcnetwork/care_livekit_fe,ohcnetwork/care_scribe_fe"
`

NOTE: Cannot update non arbitrary fields like symptoms, diagnoses for now.

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [x] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Release Notes

- **New Features**
	- Introduced keyboard shortcut rendering functionality.
	- Added a custom hook for observing changes to DOM attributes, enhancing component interactivity.
	- Implemented a voice recording hook with improved microphone permission handling.
	- Enhanced date input component to reflect external value changes.
	- Improved dropdown components with better value synchronization and accessibility features.
	- Updated audio capture dialog to utilize a new voice recorder hook.
	- Enhanced functionality in the SymptomsBuilder component for better user interaction.
	- Updated DailyRounds component for improved form handling and validation.

- **Bug Fixes**
	- Adjusted unselect radio input handling for better data representation.
	- Refined validation and threshold handling in text input fields.

- **Documentation**
	- Expanded localization support with additional key-value pairs for improved user feedback.

- **Chores**
	- Removed deprecated API routes related to the Scribe functionality.
	- Removed the Scribe component and associated functionalities.

### CODE REVIEW

**Comment by bodhish** [comment]
@shivankacker please close pending PR's before opening others.

**Comment by shivankacker** [comment]
@bodhish working on this currently

File: package-lock.json
Comment (rithviknishad) [inline package-lock.json:1]: scribe seems to be added in the lockfile 👀

**Comment by shivankacker** [comment]
Will merge conflicts once PR is tested and approved

**Comment by nihal467** [comment]
@shivankacker can you clear the merge conflict, to get it ready for review

**Comment by shivankacker** [comment]
@nihal467 resolved merge conflicts, deploy preview failing due to cache issue, nothing else.

**Comment by nihal467** [comment]
image

@shivankacker the file upload is still not working

**Comment by nihal467** [comment]
Enhancement

image

- [ ] the users never knew that its a clickable tooltip
- [ ] if we reject all auto-fill, the notification shouldn't say "autofill completed" 

image

- [x] when we talk about oral care in nursing, its filling disabled field in oral issues

image

- [ ] the above two section are not auto-filling in scribe

**Comment by shivankacker** [comment]
@nihal467 enhancements can be taken up later, they will also be dependent on the plugin (no changes in care). I have added an ignore rule for oral issue.

File: src/components/Patient/DailyRounds.tsx
Comment (shivankacker) [reply src/components/Patient/DailyRounds.tsx:565]: Thanks captain obvious

**Comment by nihal467** [comment]
conditional approved it, @shivankacker take up enhancement as separate issue

**Review by rithviknishad — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**
